internal/provider/vaultradar/resource_radar_source_github_enterprise_test.go

```diff
@@ -0,0 +1,50 @@
+package vaultradar_test
+
+import (
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+	"github.com/hashicorp/terraform-provider-hcp/internal/provider/acctest"
+)
+
+func TestRadarSourceGitHubEnterprise(t *testing.T) {
+	// Requires Project already setup with Radar.
+	// Requires a Service Account with an Admin role on the Project.
+	// Requires access to a GitHub Enterprise self-managed instance.
+	// Requires the following environment variables to be set:
+	projectID := os.Getenv("HCP_PROJECT_ID")
+	githubOrganization := os.Getenv("RADAR_GITHUB_ENTERPRISE_ORGANIZATION")
+	domainName := os.Getenv("RADAR_GITHUB_ENTERPRISE_DOMAIN")
+	token := os.Getenv("RADAR_GITHUB_ENTERPRISE_TOKEN")
+
+	if projectID == "" || githubOrganization == "" || domainName == "" || token == "" {
+		t.Skip("HCP_PROJECT_ID, RADAR_GITHUB_ENTERPRISE_ORGANIZATION, RADAR_GITHUB_ENTERPRISE_DOMAIN, and RADAR_GITHUB_ENTERPRISE_TOKEN must be set for acceptance tests")
```

Comment:
@jasonpilz Seems the automated testing is on, and this will fail for CI.
* Requires Project already setup with Radar.
* Requires a Service Account with an Admin role on the Project.
* Requires access to a GitHub Enterprise self-managed instance.

This works on my local setup with the proper env setup.

Reply:
I created the following ticket to try an address this in the future.
https://hashicorp.atlassian.net/browse/VAULT-31051

Reply:
Sounds good, thanks 👍🏻